code-of-conduct.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Code of Conduct
+layout: default
+---
+
+**<font color=red>[DRAFT!]</font>**
+
+### Research Support Network - Expert Volunteer List - Code of Conduct
+
+#### Code of Conduct (Summary View)
+
+The Research Support Network is dedicated to providing a welcoming and supportive environment for all people, regardless of background or identity. By signing up to the Expert Volunteer List or by using it to contact an expert volunteer, participants accept to abide by this Code of Conduct and accept the procedures by which any Code of Conduct incidents are resolved. Any form of behaviour to exclude, intimidate, or cause discomfort is a violation of the Code of Conduct. In order to foster a positive and professional environment we encourage the following kinds of behaviours in all interactions:
+
+* Use welcoming and inclusive language
+* Be respectful of different viewpoints and experiences
+* Gracefully accept constructive criticism
+* Show courtesy and respect towards others
+
+If you believe someone is violating the Code of Conduct, we ask that you report it to the list's maintainers via email to [expertvolunteerlist@gmail.com](expertvolunteerlist@gmail.com). Expert volunteers who are reported to have violated the Code of Conduct will be removed from the Expert Volunteer List without investigation or notice, as the list is run purely as an "address book" and the list's maintainers are not in a position to investigate reported violations or to mediate conflicts within the private interactions between expert volunteers and scientists seeking support.
+
+#### Code of Conduct (Detailed View)
+
+##### Part 1. Introduction
+
+The Research Support Network is led by a small number of computational scientists and runs the Expert Volunteer List. Anyone can sign up to the list and contributions to the list's open source backend via GitHub are welcome. We value the involvement of everyone contributing and utilizing the list and are committed to creating a friendly and respectful environment for anyone participating in list-related activities. All participants are expected to show respect and courtesy to others.
```

Comment:
"Anyone can sign up" > change to: _We encourage all computational experts to sign up for virtual support and advice for scientists forced to refocus on computational work due to a crisis if they have the capacity to do so_.